cmd/terraform/linter/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+func main() {
+	tfDirectory := flag.String("tf-directory", "", "The directory that contains the terraform files to lint")
+	flag.Parse()
+
+	if valid, err := isValidDirectory(*tfDirectory); !valid {
+		log.Fatal(err)
+	}
+
+	files, err := ioutil.ReadDir(*tfDirectory)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = LintDirectory(*tfDirectory, files)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func isValidDirectory(path string) (bool, error) {
```

Comment:
No need for `bool` as consuming code can use `err != nil` as a check for if the directory is valid.